src/channel_pool.rs

```diff
@@ -44,10 +45,16 @@ impl ChannelPool {
             },
         };
 
+        let rust_version = version_meta().unwrap().semver.to_string();
```

Comment:
What do we need the Rust version for at the server side? I don't think we need that.

Reply:
This is for the sake of consistency with python client, that now does send python version alongside the client version.

Reply:
I don't think it adds any value for Rust, as having a different Rust version should not change the way the client works, and it would only be usable to fingerprint clients. A range of Rust versions can be enforced in the Rust qdrant-client crate itself if ever needed.

Based on the above I'd prefer to remove it. That'll allow us to remove the `rustc_version` dependency as well.

@generall Thoughts?

Reply:
let's remove